dbt/adapters/firebolt/connections.py

```diff
@@ -180,56 +151,18 @@ def cancel(self, connection):
 
     @classmethod
     def get_status(cls, cursor):
```

Comment:
we don't need this method at all now actually. it was deprecated in dbt-core 0.19 almost a year ago

Reply:
Done.